lib/os/zvfs/Kconfig

```diff
@@ -33,4 +33,27 @@ config ZVFS_EVENTFD_MAX
 
 endif # ZVFS_EVENTFD
 
+config ZVFS_POLL
+	bool "ZVFS poll"
+	select POLL
+	help
+	  Enable support for zvfs_poll().
+
+if ZVFS_POLL
+
+config ZVFS_POLL_MAX
```

Comment:
Don't we need to deprecate `NET_SOCKETS_POLL_MAX` along with this patch? The config still exists, is being set by several samples/tests, but is no longer effective. I recall we've discussed this during the previous attempt: https://github.com/rlubos/zephyr/commit/6c77edc20519439d5305f678582c13c017172e40

Reply:
Would it be OK if I cherry-pick your commit?

Reply:
K - cherry-picked - I'm hoping CI doesn't complain 😅

Reply:
sure, no prob, not sure though if it's not outdated. I guess CI will tell us